### DISCUSSION TITLE
Unable to install Airflow 2.0.2 using conda-forge

### DISCUSSION THREAD

**Discussion by gastonlau** [apache/airflow#23743] [General] [answered]
### Apache Airflow version

2.0.2

### What happened

I was not able to install Airflow 2.0.2 to my EC2 instance using conda-forge starting from yesterday evening. The installation was successful earlier on the same day. Here is the complete command output :
```
conda install -qy -c conda-forge apache-airflow=2.0.2
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... Killed
```
Sometimes the last line is `*** picosat: out of memory in 'resize'` instead.

### How to reproduce

Running a bash script:
```
eval "$(conda shell.bash hook)"
conda create -q -yn $CONDA_ENV_NAME python=3.8
conda activate $CONDA_ENV_NAME
pip3 install -q --upgrade pip3
pip3 install -qU setuptools --ignore-installed
conda install -qy -c conda-forge airflow=2.0.2
```

### Operating System

Amazon Linux 2

### Deployment

Virtualenv installation

### Deployment details

I am using the AWS CodeDeploy to deploy the DAG files and airflow.config as well as other necessary components to an AWS EC2 instance.

### Anything else

 I am still able to install  package `apache-airflow-providers-slack=3.0.0` but it will automatically install Airflow 2.2.3 and other packages for me which I don't want.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment, marked as answer]
The only supported way of installing airflow is using `pip` and constratins. https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html. 

Please switch to using it, if you want to to have consistently working installation.
